lib/views/name.erb

```diff
@@ -0,0 +1 @@
+<h1>"Johan" <%= params[:name]  %> </h1>
```

Comment:
This is printing "Johan" and should throw an error for `params[:name]` because you are *not* storing anything in a variable called `:name`. In the route, you are creating a variable `:Johan`. You theoretically could call `params[:Johan]` but instead you want to make the route `/:name` -> `:name` is a variable accessible in `params`.